gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -1258,4 +1261,20 @@ public int getDatabasePort() {
   public String getDatabaseName() {
     return get(GATEWAY_DATABASE_NAME, "GATEWAY_DATABASE");
   }
+
+  @Override
+  public boolean isDatabaseSslEnabled() {
```

Comment:
So, it seems that we are assuming that we will only ever have one database - which likely makes perfect sense. This does however imply that either we will only have one JDBC TSS per deployment or that the schema will disambiguate which topology the tokens are associated with. This is something that we need to reconcile though not necessarily now.

Reply:
Thanks, @lmccay for your review! I agree that per topology may be necessary once. Let's cross that bridge when we come to it!